<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-6KPONN4H" ignored="6" name="regression exclude smoke testing" tests="3" failures="1" timestamp="2021-04-30T11:16:02 IDT" time="48.454" errors="0">
  <testcase name="loginNegative" time="21.665" classname="org.example.tests.LoginTests">
    <failure type="java.lang.AssertionError" message="Different error message did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Different error message did not expect to find [true] but found [false]
at org.example.tests.LoginTests.loginNegative(LoginTests.java:160)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- loginNegative -->
  <testcase name="loginNegative" time="2.134" classname="org.example.tests.LoginTests"/>
  <testcase name="loginNegative" time="2.172" classname="org.example.tests.LoginTests"/>
  <testcase name="negativePasswordIncorrect" time="0.0" classname="org.example.tests.LoginTests">
    <ignored/>
  </testcase> <!-- negativePasswordIncorrect -->
  <testcase name="loginNegativeLoginIncorrect2" time="0.0" classname="org.example.tests.LoginTests">
    <ignored/>
  </testcase> <!-- loginNegativeLoginIncorrect2 -->
  <testcase name="loginNegativeParametricThird" time="0.0" classname="org.example.tests.LoginTests">
    <ignored/>
  </testcase> <!-- loginNegativeParametricThird -->
  <testcase name="loginNegativeParametricSecond" time="0.0" classname="org.example.tests.LoginTests">
    <ignored/>
  </testcase> <!-- loginNegativeParametricSecond -->
  <testcase name="loginPositive" time="0.0" classname="org.example.tests.LoginTests">
    <ignored/>
  </testcase> <!-- loginPositive -->
  <testcase name="loginNegativeLoginEmpty" time="0.0" classname="org.example.tests.LoginTests">
    <ignored/>
  </testcase> <!-- loginNegativeLoginEmpty -->
</testsuite> <!-- regression exclude smoke testing -->
